contracts/Cashier.sol

```diff
@@ -826,28 +841,158 @@ contract Cashier is ICashier, UsingHelpers, ReentrancyGuard, Ownable, Pausable {
+        }
+
+        if (paymentType == TKNTKN) {
+            tokenAddress = IVoucherKernel(voucherKernel).getVoucherPriceToken(
+                tokenSupplyId
+            );
+
+
+            //Reduce _from escrow token amount and increase _to escrow token amount - price 
+            escrowTokens[tokenAddress][_from] =  escrowTokens[tokenAddress][_from].sub(price);
+            escrowTokens[tokenAddress][_to] =  escrowTokens[tokenAddress][_to].add(price);
+
+            tokenAddress = IVoucherKernel(voucherKernel).getVoucherDepositToken(
+                tokenSupplyId
+            );
+
+            //Reduce _from escrow token amount and increase _to escrow token amount - deposit 
+            escrowTokens[tokenAddress][_from] =  escrowTokens[tokenAddress][_from].sub(depositBu);
+            escrowTokens[tokenAddress][_to] =  escrowTokens[tokenAddress][_to].add(depositBu);
+
+        }
+   
+        IBosonRouter(bosonRouterAddress).incrementCorrelationId(_to);
     }
 
+    /**
+     * @notice After the transfer happens the _tokenSupplyId should be updated in the promise. Escrow funds for the seller's deposits (If in ETH) should be allocated to the new owner as well.
+     * @param _from prev owner of the _tokenSupplyId
+     * @param _to nex owner of the _tokenSupplyId
+     * @param _tokenSupplyId _tokenSupplyId for transfer
+     * @param _value qty which has been transferred
+     */
+    function onERC1155Transfer(
+        address _from,
+        address _to,
+        uint256 _tokenSupplyId,
+        uint256 _value
+    ) external override onlyTokensContract {
+        uint8 paymentType =
+            IVoucherKernel(voucherKernel).getVoucherPaymentMethod(
+                _tokenSupplyId
+            );
+
+        uint256 depositSe;
+        uint256 totalAmount;
+
+        if (paymentType == ETHETH || paymentType == TKNETH) {
+            depositSe = IVoucherKernel(voucherKernel).getSellerDeposit(
+                _tokenSupplyId
+            );
+            totalAmount = depositSe.mul(_value);
+
+            //Reduce _from escrow amount and increase _to escrow amount
+            escrow[_from] = escrow[_from].sub(totalAmount);
+            escrow[_to] = escrow[_to].add(totalAmount);
+        }
+
+        if (paymentType == ETHTKN || paymentType == TKNTKN) {
+            address tokenDepositAddress =
+                IVoucherKernel(voucherKernel).getVoucherDepositToken(
+                    _tokenSupplyId
+                );
+
+            depositSe = IVoucherKernel(voucherKernel).getSellerDeposit(
+                _tokenSupplyId
+            );
+            totalAmount = depositSe.mul(_value);
+
+            //Reduce _from escrow token amount and increase _to escrow token amount - deposit
+            escrowTokens[tokenDepositAddress][_from] =  escrowTokens[tokenDepositAddress][_from].sub(totalAmount);
+            escrowTokens[tokenDepositAddress][_to] =  escrowTokens[tokenDepositAddress][_to].add(totalAmount);
```

Comment:
Two whitespaces after `=`
Please run the pipeline locally, so we avoid that.

Reply:
I did run the pipeline locally -- multiple times. I ran the ./go script and also ./go contracts:lint, ./go contracts:lint_fix, ./go contracts:format, ./go contracts:format_fix, ./go tests:lint, ./go tests:lint_fix, ./go tests:format, and ./go tests:format_fix separately.

As you can see, it also passed when the pipeline was run from GitHub. This was never flagged. I will change it manually, but the scripts don't flag everything so some things can slip through.